techniques/general/G199.html

```diff
@@ -2,120 +2,99 @@
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content that accepts user data input.</p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p>利用者のデータ入力を受け入れているコンテンツ</p>
+            <p>これは、次の達成基準に関する達成方法である:</p>
             <ul>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-identification">Success Criterion 3.3.1: Error Identification</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-identification">達成基準 3.3.1: エラーの特定</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-suggestion">Success Criterion 3.3.3: Error Suggestion</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-suggestion">達成基準 3.3.3: エラー修正の提案</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-prevention-legal-financial-data">Success Criterion 3.3.4: Error Prevention (Legal, Financial, Data)</a> (Advisory)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/error-prevention-legal-financial-data">達成基準 3.3.4: エラー回避 (法的、金融、データ)</a> (参考)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/status-messages">Success Criterion 4.1.3: Status Messages</a> (Sufficient as a way to meet <a href="../aria/ARIA22">ARIA22: Using role=status to present status messages</a>)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/status-messages">達成基準 4.1.3: ステータスメッセージ</a> (<a href="../aria/ARIA22">ARIA22: ステータスメッセージを提示するために role=status を使用する</a>の達成方法として十分)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to reduce the effort required for users to confirm
-               that an action, such as submitting a Web form, was completed successfully. This can
-               be accomplished by providing consistently presented feedback that explicitly indicates
-               success of an action, rather than requiring a user to navigate through content to
-               discover if the action was successful.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、利用者にアクションを確認させる手間 (例えば、ウェブフォームを送信した時に、それが正常に完遂されたことを利用者に確認させる) をなくすことである。アクションが成功したかを確認させるように促す代わりに、アクションが成功したか明確に示すフィードバックを常に提供することで達成することができる。
             </p>
-            <p>Significant effort can be expended by users who can not easily scan through information
-               to confirm their action (such as that data submitted has been successfully entered
-               into a database, sent to a person, or added to content being edited).
+            <p>アクションを確認するために、情報に目を通すことが容易ではない利用者の大きな手間が費やされる時がある (例えば、データベースにデータが正常に送信されたか、人に送られたのか、または編集されているコンテンツに追加されたのか)。
```

Comment:
例示ですので、「または」を除いてみました。
```suggestion
            <p>アクションを確認するために、情報に目を通すことが容易ではない利用者の大きな手間が費やされる時がある (例えば、データベースにデータが正常に送信された、人に送られた、編集されているコンテンツに追加された)。
```